### ISSUE TITLE
No sense parsing for {{ru-verb-alt-ё}} template

### ISSUE THREAD

**Issue by jsannemo** [tatuylonen/wiktextract#131]
See e.g. https://en.wiktionary.org/wiki/%D0%B5%D1%80%D0%B7%D0%B0%D1%82%D1%8C

that receives no sense here: https://kaikki.org/dictionary/All%20languages%20combined/meaning/%D0%B5/%D0%B5%D1%80/%D0%B5%D1%80%D0%B7%D0%B0%D1%82%D1%8C.html

**Comment by tatuylonen** [comment]
This was caused by a rather unusual template construction that produces both word head and its senses.  I fixed it by manually listing such exceptional templates in additional_expand_templates (in wiktextract/page.py).  I also found a few other similar templates: ru-noun-alt-ё, ru-adj-alt-ё, ru-proper noun-alt-ё, ru-pos-alt-ё.  These are now also expanded before parsing.

The changes should be reflected on https://kaikki.org within a couple of days.
